gliderpy/fetchers.py

```diff
@@ -47,7 +59,9 @@ def __init__(self, server=_server):
             server=server,
             protocol="tabledap",
         )
-        if "ifremer" in self.fetcher.server:
+        if "oceanobservatories" in self.fetcher.server:
+            self.fetcher.variables = ooi_vars
+        elif "ifremer" in self.fetcher.server:
```

Comment:
If we go for the dictionary instead of multiple if-clauses we'll have something like `self.fetcher.variables = server_vars[server]` here.

Reply:
Much more elegant! Addressed in a667bc8